wp-content/themes/haruoshi_origin/single-works.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * This is single-works.php
+ *
+ * @package haruoshi-origin
+ */
+
+get_header(); ?>
+
+<article class="container">
+	<h1 class="container__title"><?php the_title(); ?></h1>
+	<p class="container__date"><?php the_time( 'Y/m/d' ); ?></p>
```

Comment:
```suggestion
	<time class="container__date"><?php the_time( 'Y/m/d' ); ?></time>
```
こっちのほうがマークアップとしてきれいです。

Reply:
done https://github.com/oooshima/origin_wordpress/pull/28/commits/729d01774cb4a2581699a13e47c97b63ca58f6f8